story_time/channel_introduction.py

```diff
@@ -8,14 +8,14 @@ def channel_introduction(role):
 **WELCOME, ASSASSIN**
 **=========================**
 This town is like an old kitchen... it's a giant mess. Luckily, the town has *you*, \
-their protector, who picks out the cockroaches for them and squishes them like the puny animals they are. 
+their protector, who picks out the cockroaches for them and squishes them like the puny animals they are.
 You do not exactly know what is about to happen, but your senses never lie; \
 something *TERRIBLE* is happening, and it depends on your actions how much the town will suffer from it!
-You do not know your allies, your masters taught you never to trust anyone. However, \
+You do not know your allies- your masters taught you never to trust anyone. However, \
```

Comment:
If you want to put a - in-between, you gotta put a space in front.

Reply:
Got it. Thanks!